src/statsmodels/statsmodel.jl

```diff
@@ -86,6 +86,16 @@ function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame;
     return(out)
 end
 
+# Predict function with confidence intervars
+function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame, interval_type::Symbol, level::Real = 0.95; kwargs...)
```

Comment:
You could merge the two methods by having `interval_type::Union{Symbol, Void}=nothing`. Then you would check  `interval_type == nothing` inside the body (ideally the method in `StatsBase` would also accept `nothing` so that you can even pass it directly).